src/app/app.component.ts

```diff
@@ -53,6 +53,9 @@ export class AppComponent implements OnInit {
       if (readySource === 'cordova') {
         this.insomnia.keepAwake();
       }
+      if (this.platform.is('ios')) {
```

Comment:
I think this should be moved to the Speech constructor in `src/core/speech.ts`; also a comment like
```
// See https://github.com/vilic/cordova-plugin-tts/issues/40
```
would make sense for reference.